reactor-netty-core/src/main/java/reactor/netty/transport/NameResolverProvider.java

```diff
@@ -177,6 +178,13 @@
 		 */
 		NameResolverSpec resolvedAddressTypes(ResolvedAddressTypes resolvedAddressTypes);
 
+		/**
+		 * Configure the address that will be used to bind too. If `null` the default will be used.
+		 * @param localAddress the bind address
+		 * @return {@code this}
+		 */
+		NameResolverSpec localAddress(SocketAddress localAddress);
```

Comment:
Let's rename the method and accept a `Supplier` as in the rest of the `Transport`s
`bindAddress(Supplier<? extends SocketAddress> bindAddressSupplier)`